Sources/StreamVideoSwiftUI/CallView/VideoRenderer.swift

```diff
@@ -17,36 +17,38 @@ public struct LocalVideoView<Factory: ViewFactory>: View {
     private var participant: CallParticipant
     private var idSuffix: String
     private var call: Call?
-    
+    private var availableFrame: CGRect
+
     public init(
         viewFactory: Factory,
         participant: CallParticipant,
         idSuffix: String = "local",
         callSettings: CallSettings,
-        call: Call?
+        call: Call?,
+        availableFrame: CGRect
```

Comment:
We should document all the public api changes in the changelog.

Reply:
Definitely, just one step at a time. I want to get the PR passing and feedback addressed so there are no further API changes and i'll update it.